src/API/Data/PeopleRepository.cs

```diff
@@ -18,33 +18,48 @@ public class PeopleRepository : DataRepository
+                                || EF.Functions.ILike(p.Name, $"%{query.Q}%")
+                                || (string.IsNullOrWhiteSpace(parsedName.firstName) == false
+                                    && string.IsNullOrWhiteSpace(parsedName.lastName) == false
+                                    && EF.Functions.ILike(p.Name, $"{parsedName.firstName}%{parsedName.lastName}%")))
+                        .Where(p => // check for overlapping responsibilities / job classes
                             query.Responsibilities == Responsibilities.None
                                 || ((int)p.Responsibilities & (int)query.Responsibilities) != 0)
                                 // That & is a bitwise operator - go read-up!
                                 // https://stackoverflow.com/questions/12988260/how-do-i-test-if-a-bitwise-enum-contains-any-values-from-another-bitwise-enum-in
-                        .Where(p=> // partial match any supplied interest against any self-described expertise
+                        .Where(p => // partial match any supplied interest against any self-described expertise
                             query.Expertise.Length == 0
-                                || query.Expertise.Select(s=>$"%{s}%").ToArray().Any(s => EF.Functions.ILike(p.Expertise, s)))
-                        .Where(p=> // partial match campus
+                                || query.Expertise.Select(s => $"%{s}%").ToArray().Any(s => EF.Functions.ILike(p.Expertise, s)))
+                        .Where(p => // partial match campus
                             query.Campus.Length == 0
-                                || query.Campus.Select(s=>$"%{s}%").ToArray().Any(s => EF.Functions.ILike(p.Campus, s)))
+                                || query.Campus.Select(s => $"%{s}%").ToArray().Any(s => EF.Functions.ILike(p.Campus, s)))
                         .Where(p => query.Roles.Length == 0
-                                || p.UnitMemberships.Any(m => query.Roles.Contains(m.Role) && m.Unit.Active))
+                            || p.UnitMemberships.Any(m => query.Roles.Contains(m.Role) && m.Unit.Active))
                         .Where(p => query.Permissions.Length == 0
-                                || p.UnitMemberships.Any(m => query.Permissions.Contains(m.Permissions) && m.Unit.Active))
+                            || p.UnitMemberships.Any(m => query.Permissions.Contains(m.Permissions) && m.Unit.Active))
                         .Include(p => p.Department)
                         .AsNoTracking()
                         .ToListAsync();
                     return Pipeline.Success(result);
                 });
 
-        private static IQueryable<Person> GetPeopleFilteredByArea(PeopleContext db, PeopleSearchParameters query) 
+        private static (string firstName, string lastName) GetParsedName(string nameQuery)
+        {
+            if (string.IsNullOrWhiteSpace(nameQuery) == false && nameQuery.Count(q => q == ',') == 1)
```

Comment:
My only quibble is that if a nameQuery starts with a comma, eg. `, Jason` a 500 error is thrown.  We could handle this case, or ignore it.  To the user it just looks like there are no results.